### ISSUE TITLE
PHP Fatal error:  Cannot declare class OCA\Assistant\Migration\Version010003Date20231208083112, because the name is already in use in

### ISSUE THREAD

**Issue by marcelklehr** [nextcloud/assistant#87]
### Which version of assistant are you using?

latest

### Which version of Nextcloud are you using?

28

### Which browser are you using? In case you are using the phone App, specify the Android or iOS version and device please.

-

### Describe the Bug

When updating the app I get

```
PHP Fatal error:  Cannot declare class OCA\Assistant\Migration\Version010003Date20231208083112, because the name is already in use in /opt/nextcloud/28.0.0/apps/assistant/lib/Migration/Version010003Date20231208083112.php on line 16
```

### Expected Behavior

No error

### To Reproduce

Update app

**Comment by julien-nc** [comment]
I don't think we are doing anything wrong here. It only happens in your setup. Could you restart php-fpm if you're using it?

**Comment by marcelklehr** [comment]
Seemed like an autoloader thing, it happened in production on zitsh

**Comment by julien-nc** [comment]
"Seemed" so has the migration step ran eventually?

**Comment by marcelklehr** [comment]
Idk, nothing seems broken though
